app/components/emoji_picker/emoji_picker.js

```diff
@@ -220,22 +314,37 @@ class EmojiPicker extends PureComponent {
                     />
                 </View>
                 <View style={styles.container}>
-                    <SectionList
-                        showsVerticalScrollIndicator={false}
-                        style={styles.listView}
-                        sections={emojis}
-                        renderSectionHeader={this.renderSectionHeader}
-                        renderItem={this.renderItem}
-                        removeClippedSubviews={true}
-                    />
+                    {listComponent}
+                    {!searchTerm &&
+                        <View style={styles.bottomContentWrapper}>
+                            <View style={styles.bottomContent}>
+                                {this.renderSectionIcons()}
```

Comment:
perhaps this would be best as the footer of the sectionList

Reply:
Didn't have good results with putting this in the footer. I adjusted the bottomMargin on the listview instead.